doc/docs/master/how-tos/developer-workflow/build-pipelines.md

```diff
@@ -0,0 +1,119 @@
+# Build Pipelines
+
+A build pipeline is a useful feature when iterating on the code in your pipeline. They allow you to bypass the Docker build process and submit your code directly to the pipeline. In essence, build pipelines automate Steps 2-4 of the [pipeline workflow](working-with-pipelines.md). A diagram of the build pipeline process is shown below.
```

Comment:
In some ways it removes the need for many of those steps.

"In essence, build pipelines automate or remove the need for Steps 2-? of the [pipeline workflow](working-with-pipelines.md)"

Reply:
What do you think about this? 
```suggestion
A build pipeline is a useful feature when iterating on the code in your pipeline. In essence, build pipelines automate or remove the need for Steps 2-4 of the [pipeline workflow](working-with-pipelines.md). They allow you to bypass the Docker build process and submit your code directly to the pipeline. A diagram of the build pipeline process is shown below.
```